umbraco/Umbraco-CMS
# v9: appsettings.json now generate from template if missing

# Notes
- Renamed `appsettings.json` to `appsettings.template.json`
- Renamed `appsettings.Development.json` to `appsettings.Development.template.json`
- Implemented 2 new targets in Umbraco.Web.UI.Netcore.csproj that will copy and rename the respective files.

# How to test
- From a clean install: 
- go to your  "Umbraco.Web.UI.Netcore" folder, it should be missing an appsettings.json & appsettings.Development.json file
- Build the project
- Files should now be generated

- From an already built application:
- go to your  "Umbraco.Web.UI.Netcore" folder
- (If you want to save ur changes like connection string, remember to copy it into the new template file)
- Delete appsettings.json and appsettings.Development.json
- Build the project
- Files should now be generated
